gulpfile.js

```diff
@@ -4,8 +4,7 @@ const sass = require('gulp-sass');
 const del = require('del');
 sass.compiler = require('node-sass');
 nodemon = require('gulp-nodemon');
-const tsConfigFileName = 'tsconfig.json';
-const tsProject = ts.createProject(tsConfigFileName);
+const tsProject = ts.createProject('tsconfig.prod.json');
```

Comment:
what do you mean by 'prod' .json?

Reply:
We need separate tsconfig file that excludes test files otherwise we would have compiled and copied test resources into `dist` folder.